src/gulpfile.js

```diff
@@ -255,6 +255,29 @@ gulp.task('build-html', ['add-revision-numbers'], function () {
   .pipe(gulp.dest(paths.build));
 });
 
+gulp.task('build-mseditor', ['template-editor', 'add-revision-numbers'], function () {
```

Comment:
If I am not wrong, as it is right now, `template-editor` is a facto dependency of `add-revision-numbers`.

What do you think about adding as dependency explicitly? or, another alternative is apply the revision numbers independently of the other scripts.

Reply:
I like the first option, this task it is being used to do the revision on all the build , I like continue using it than create a parallel one, thanks!